src/lib.rs

```diff
@@ -7,7 +7,7 @@ mod enums;
 mod serialize;
 mod structs;
 
-#[proc_macro_derive(Serialize)]
+#[proc_macro_derive(Serialize, attributes(edn))]
```

Comment:
I was testing and for performance reasons I would think of creating another derive macro like `SerializeWith` or `SerializeAttr` that contains the attribute macros code. It becomes way slower with those nested `fors`.

Reply:
Really? But performance at compile time right?

Reply:
it took 12% more out of performance